### PR TITLE
Music releases - Petra Brinck 

### PR DESCRIPTION
**Pull request by petrabrinck**
Link to my deployed project: https://brave-jones-f321b1.netlify.com

My reflections on how this week's project turned out:
It was challenging to get all the details in place but I finally saw some ”clusters" of code and began to get a little better understanding when it comes to React. I tend to make things more complicated than they could be so in the future I will try to find easier solutions with less code.  I forgot mobile first so despite help today I still can’t get media queries to work the way I want them to. I will continue to try to solve it :-) 
I do think It was a great project though. Very inspiring and relevant since we went to an meetup at Spotify and got to meet and listen to several female speakers (all working there) talk about their daily job and their journeys towards becoming developers. 

Things I'd like to have clarified or explained in more detail:
How do I go about doing media queries when I forget to do mobile first?
Thanks! 
Petra

### CODE REVIEW

File: code/src/Album.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import './components/album.css';
+import './components/icons.css';
+/* import data from './data.json'*/
+
+/*console.log(data)
+console.log(data.album.items[1])*/
+
+export const Album = (props) => {
+    return (
```
Comment (idlefingers) [inline code/src/Album.js]: Looks like your editor is set up to do 4 spaces for indentation (should be 2). See [this question on stack overflow](https://stackoverflow.com/c/technigo/questions/253) for a bunch of things to check.

File: code/src/Album.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                </div>
+            </div>
+        </div>
+    )
+}
+
+
+
+
+
```
Comment (idlefingers) [inline code/src/Album.js]: Try to avoid having lots of blank lines in your code - just to keep things tidy!

File: code/src/Artists.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+export const Artists = () => {
+    return (
+        <div>
+            <data />
+            Export Artists!
+        </div>
+    )
+}
+
```
Comment (idlefingers) [inline code/src/Artists.js]: Looks like this file isn't used (and react doesn't know how to render json, so it'd break if it was!). Don't forget to remove unused code.

File: code/src/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (idlefingers) [inline code/src/index.html]: This file isn't needed - the one in the public folder is the one which is used.

**Comment by petrabrinck** [comment]
Thank you ever so much Damien! 
I will go thru all comments carefully and update (if not soon, I'll do it over Christmas :-)

**Pull request closed without merging**